pkgs/applications/version-management/gitqlient/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, mkDerivation, fetchFromGitHub
+, qtbase, qmake
+}:
+
+with stdenv.lib;
+
+mkDerivation rec {
+  pname = "GitQlient";
+  version = "1.1.0-beta";
+
+  src = fetchFromGitHub {
+    owner = "francescmm";
+    repo = "GitQlient";
+    rev = "v${version}";
+    sha256 = "1x3i2w2xdvq7fknrpdcqsmf8bp9s9w8hyw2f3c3fvpyrhmm5ck28";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [ qtbase ];
+  nativeBuildInputs = [ qmake ];
```

Comment:
Not sure if these 2 lines are required when you use `libsForQt5` in `all-packages.nix`.

Reply:
The first one is not needed, but the second one is required.

Reply:
As of 1.3.0, qtwebengine is required.